packages/venia-concept/src/components/Gallery/item.js

```diff
@@ -73,7 +73,7 @@ class GalleryItem extends Component {
                     {this.renderImage()}
                 </Link>
                 <Link to={resourceUrl(productLink)} className={classes.name}>
-                    <span>{name}</span>
+                    <span dangerouslySetInnerHTML={{ __html: name }} />
```

Comment:
This is not the right approach. It's a vector for XSS, and it doesn't reflect the purpose of this node (`name` should be a `utf-8` string, not HTML).

Reply:
We could probably close this PR and the associated issue then, right? Seems like it isn't a real bug if the data should be a `utf-8` string already.